tests/unit/test_query_headers.py

```diff
@@ -74,3 +74,9 @@ def test_no_query_comment_on_msck(self):
         self.query_header.comment.append = True
         sql = "MSCK REPAIR TABLE"
         assert self.query_header.add(sql) == sql
+
+    def test_no_query_comment_on_vacuum_with_leading_whitespaces(self):
```

Comment:
could you add also another test with lowercase vacuum and/or optimize
e.g.
`sql = vacuum table"`
looks like that we don't have any test case with lowercase string, I just want to have one for that - thanks

Reply:
Added ✅